features/mbedtls/importer/adjust-config.sh

```diff
@@ -37,36 +37,41 @@ add_code() {
 
 # add an #ifndef to include config-no-entropy.h when the target does not have
 # an entropy source we can use.
-add_code                                                                                  \
-    "#ifndef MBEDTLS_CONFIG_H\n"                                                          \
-    "\n"                                                                                  \
-    "#include \"platform\/inc\/platform_mbed.h\"\n"                                       \
-    "\n"                                                                                  \
-    "\/*\n"                                                                               \
-    " * Only use features that do not require an entropy source when\n"                   \
-    " * DEVICE_ENTROPY_SOURCE is not defined in mbed OS.\n"                               \
-    " *\/\n"                                                                              \
-    "#if !defined(MBEDTLS_ENTROPY_HARDWARE_ALT) && !defined(MBEDTLS_TEST_NULL_ENTROPY)\n" \
-    "#include \"mbedtls\/config-no-entropy.h\"\n"                                         \
-    "\n"                                                                                  \
-    "#if defined(MBEDTLS_USER_CONFIG_FILE)\n"                                             \
-    "#include MBEDTLS_USER_CONFIG_FILE\n"                                                 \
-    "#endif\n"                                                                            \
-    "\n"                                                                                  \
+add_code                                                                \
+    "#ifndef MBEDTLS_CONFIG_H\n"                                        \
+    "\n"                                                                \
+    "#include \"platform\/inc\/platform_mbed.h\"\n"                     \
+    "\n"                                                                \
+    "\/*\n"                                                             \
+    " * Only use features that do not require an entropy source when\n" \
+    " * DEVICE_ENTROPY_SOURCE is not defined in mbed OS.\n"             \
```

Comment:
@andresag01 Where is `DEVICE_ENTROPY_SOURCE` defined? It doesn't occur in the mbed-OS repository (I only found `DEVICE_TRNG` in relation to entropy). Depending on its definition, it might need to be updated to incorporate the new potential source of entropy from the NV seed.

Reply:
This is simply an out-of-date comment from PR https://github.com/ARMmbed/mbed-os/pull/2716. I think my original idea was to create a simple macro that would make it easy to tell if an entropy source is available. However, I think this idea was scraped at some point (due to some complications with config/implementation) and unfortunately the comment was not updated.

Note that I did not change this line in the PR other than to align the \ character. I will update the comment before removing the DRAFT label. However, could you please look at the actual code changes for the time being? Thanks!